pkg/token/options.go

```diff
@@ -0,0 +1,47 @@
+package token
+
+import "github.com/Azure/kubelogin/pkg/internal/token"
+
+// list of supported login methods for library consumers
+
+const (
+	ServicePrincipalLogin = token.ServicePrincipalLogin
+	ROPCLogin             = token.ROPCLogin
```

Comment:
i think we should drop ropc unless someone requested it

Reply:
done, dropped the `ROPCLogin` method